### ISSUE TITLE
Fail to train customed data using train_deep_ls.py

### ISSUE THREAD

**Issue by fangchuan** [Kamysek/DeepLocalShapes#12]
Hi, thank you for release this fantastic work, I adapt your codes to train my data(warp from iGibson_obj). Everything is fine until I went into the train_deep_ls.py, It seems that python fail to handle DataLoader or anyother variable related to gradient when using multiprocess toolkit. 
https://github.com/Kamysek/DeepLocalShapes/blob/24ee92889381d40acbb5ad1c7c8abb512a8c26b5/train_deep_ls.py#L546-L560

I havnt figure out how to solve this problem, could you help me? Please. @Kamysek @Freephi 

There is the printed logging message:
![image](https://user-images.githubusercontent.com/8519718/197448497-b736493d-2358-483b-b799-4c184f2f02a6.png)
![image](https://user-images.githubusercontent.com/8519718/197448530-9e83a3a0-eb1d-4796-913b-c9c9a5d0743a.png)

My server used in this experiment is configured as below:
python: 3.6.13
torch: 1.4.0
cuda: 10.1
os: ubuntu 18.04

**Comment by sunyuanfu** [comment]
Hello.I wondr if you have solved your problem? I have trouble in setting up the environment too.
